webapp/src/ts/services/enketo.service.ts

```diff
@@ -340,8 +340,10 @@ export class EnketoService {
           .then(valid => {
             if (valid) {
               const currentIndex = form.pages._getCurrentIndex();
-              window.history.pushState({ enketo_page_number: currentIndex }, '');
-              this.setupNavButtons($wrapper, currentIndex);
+              if (currentIndex !== undefined) {
```

Comment:
I'm usually reluctant to change app code just to overcome test limitations. Is there a way mock `_getCurrentIndex` so this change isn;t necessary?

Reply:
even if I make it return something, the `window.history` executes and karma doesn't like it

Reply:
actually it's just the `window.history.back`, 1 sec. I'm going to push a second version

Reply:
Okay, I've reverted the validation and now it just updates the test file. Pushed!
![Screen Shot 2022-09-12 at 6 30 22 pm](https://user-images.githubusercontent.com/66472237/189642723-e6c8d47e-ce38-4d7c-a870-78e23accff37.png)